packages/todomvc/package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "eslint 'src/**/*.ts' 'test/**/*.ts'",
     "test": "mocha -r ts-node/register test/**/*.test.ts",
     "build": "yarn prepack",
-    "prepack": "rm -rf dist && mkdir -p dist && cd app && cp app.package.json package.json && react-scripts build && cd .. && mv app/build dist/app && tsc --outdir dist --module commonjs --declaration --sourcemap && chmod a+x dist/start.js"
+    "prepack": "rm -rf dist && yarn mkdirp dist -p && cd app && cp app.package.json package.json && react-scripts build && cd .. && mv app/build dist/app && tsc --outdir dist --module commonjs --declaration --sourcemap && chmod a+x dist/start.js"
```

Comment:
Windows misbehaves with the `-p` and creates a folder for `-p` instead. So using the `mkdirp` node script to help us out here.